pkg/apis/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -46,6 +46,9 @@ type VPCNetworkConfigurationSpec struct {
 	// +kubebuilder:validation:MaxLength=8
 	// +optional
 	ShortID string `json:"shortID,omitempty"`
+	// Policy path of the NSX VPC the Namespace associated with.
+	// +optional
+	VPCPath string `json:"vpcPath,omitempty"`
 }
```

Comment:
You can only modify this file, and then `make manifests` will generate the new yaml files, `make generated` will copy it to `nsx.vmware.com` dir and generate other clients by code-generator.

Reply:
The latest version has performed the required make operations.